toolkit/tools/imagecustomizer/container/build-mic-container.sh

```diff
@@ -11,32 +11,16 @@ function showUsage() {
     echo "usage:"
     echo
     echo "build-mic-container.sh \\"
-    echo "    -r <container-registry> \\"
-    echo "    -n <container-name> \\"
     echo "    -t <container-tag>"
```

Comment:
should we rename this tag to something that indicates it's the full path?

Reply:
The docker CLI help has:

`  -t, --tag list                Name and optionally a tag in the "name:tag" format`

So, I think "tag" can also refer to the full path, not just the version tag.

Reply:
sounds like an overloaded term